examples/userapi/04_boundary_conditions.ipynb

```diff
@@ -50,16 +50,19 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from devito.symbolics import retrieve_functions, INT
</pre><p>Maybe points to where it's taken from.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1952/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>